provisioner/terraform/parse_test.go

```diff
@@ -205,6 +207,15 @@ func TestParse(t *testing.T) {
 		{
 			Name: "workspace-tags",
 			Files: map[string]string{
+				`main.tf`: `
+					terraform {
+					  required_providers {
+						coder = {
+						  source = "coder/coder"
+						}
+					  }
+					}
+				`,
```

Comment:
Why is this required now when it wasn't before?  Wasn't `terraform init` implicit in configuring the session? I don't get what's changed.